src/Mutator/Boolean/LogicalOr.php

```diff
@@ -77,6 +79,76 @@ public function mutate(Node $node): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\BinaryOp\BooleanOr;
+        if (!$node instanceof Node\Expr\BinaryOp\BooleanOr) {
+            return false;
+        }
+
+        $equalOp = [
+            Node\Expr\BinaryOp\Identical::class,
+            Node\Expr\BinaryOp\Equal::class,
+        ];
+
+        if (
+            in_array(get_class($node->left), $equalOp, true) === true
```

Comment:
`get_class($node->left)` - does it make sense to move to variable? As it's being called many times. Same for `$node->right`

Reply:
Yes we could. When upgrading PHP we could then use `$node->left::class` and maybe we will have to rollback the variable.

Reply:
ok, let's do it

Reply:
done